src/main/java/com/a6raywa1cher/hackservspring/service/impl/UserServiceImpl.java

```diff
@@ -145,9 +150,23 @@ public User setVendorSub(User user, VendorId vendorId, String vendorSub) {
 		return repository.findFirstByUserRole(role);
 	}
 
+	@Override
+	public User setDocumentResumePath(User user, String path) {
+		user.setDocumentResumePath(path);
+		return repository.save(user);
+	}
+
+	@Override
+	public User deleteResume(User user) {
+		discService.deleteResource(user.getDocumentResumePath());
+		user.setDocumentResumePath(null);
+		return repository.save(user);
+	}
+
 	@Override
 	@Transactional(rollbackOn = Exception.class)
 	public void deleteUser(User user) {
+		discService.deleteResource(user.getDocumentResumePath());
```

Comment:
Лучше использовать вызов `this.deleteResume(user);`. предварительно проверив на null
```suggestion
		if (user.getDocumentResumePath() != null) {
			discService.deleteResource(user.getDocumentResumePath());
		}
```